README.md

```diff
@@ -202,7 +202,7 @@ For these reasons it's necessary to have custom-built bundler plugins. Such plug
 
 ### Esbuild Support
 
-This library provides experimental esbuild support in the form of an esbuild plugin, and requires at least Node.js v14.17. To use the plugin, make sure you have `dd-trace@3+` installed, and then require the `dd-trace/esbuild` module when building your bundle.
+This library provides experimental esbuild support in the form of an esbuild plugin, and currently requires at least Node.js v16.17 or v18.7. To use the plugin, make sure you have `dd-trace@3+` installed, and then require the `dd-trace/esbuild` module when building your bundle.
```

Comment:
The support range for this changed somehow? Just a note to ensure we update the corp docs too.